assets/scss/core/_base.scss

```diff
@@ -0,0 +1,67 @@
+/*Elements form*/
+
+legend {
+  display: none;
```

Comment:
this is mean can't use legend tag?

Reply:
Si se usa por semántica y para el navegador, no para el usuario. Los navegadores que son para no videntes leen el legend y le dan contexto al formulario que el usuario esta completando. No daña tenerlo por semántica, por eso no tenemos un uso práctico y tiene un display none